module/VuFind/sql/pgsql.sql

```diff
@@ -189,6 +189,22 @@ UNIQUE (session_id)
 CREATE INDEX last_used_idx on session(last_used);
 
 --
+-- Table structure for table external_session
+--
+
+DROP TABLE IF EXISTS "external_session";
+
+CREATE TABLE external_session (
+id SERIAL,
+session_id varchar(128),
```

Comment:
Any particular reason why the varchar fields are NOT NULL in MySQL but NULLable in PostgreSQL?